openquake/calculators/base.py

```diff
@@ -702,12 +702,12 @@ def get_gmv_data(sids, gmfs):
     return numpy.fromiter(it, gmv_data_dt)
 
 
-def get_gmfs(calc):
+def get_gmfs(self):
```

Comment:
I suggest to rename the parameter, otherwise it might be counter-intuitive

Reply:
Done